addon-test-support/@ember/test-helpers/dom/trigger-key-event.js

```diff
@@ -22,7 +22,7 @@ const keyFromKeyCode = {
   18: 'Alt',
   20: 'CapsLock',
   27: 'Escape',
-  32: '',
+  32: 'Space',
```

Comment:
Why this change? I think I remember getting those values myself and being surprised by the empty string, but it was the value I saw when debugging

Reply:
I got that value from https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/keyCode 🤔

Reply:
I created this which to test it and it seems to say otherwise: https://jsbin.com/qizayecoqe/edit?js,console,output

Reply:
that is `' '`, not `''` though, so also something different. I'm fine with using `' '` instead.

Reply:
I thought I had a space, not an empty string. My bad.

Reply:
updated to use `' '` now